src/annotator/integrations/vitalsource.ts

```diff
@@ -112,6 +113,16 @@ type MosaicBookElement = HTMLElement & {
    * chapter/segment (in an EPUB-based book).
    */
   getCurrentPage(): Promise<PageInfo>;
+
+  /**
+   * Navigate the book to the page or content document whose CFI matches `cfi`.
+   */
+  goToCfi(cfi: string): void;
```

Comment:
There is a casing inconsistency between these two methods, but we don't get to decide the names.